### PR TITLE
Adding feature- print message on Installation Issue#708

### PR DESCRIPTION
**Pull request by kapilkd13**
adding message field in regard to Issue #708 . setting default message to " Successfully Installed Database!!". Enabled a new field -  message, when user choose `new_json`
This PR affects all json's. close scrutiny required.
Thanks

### CODE REVIEW

**Comment by henrykironde** [comment]
Some extra files got included. Clean your branch (git clean-xf)

**Comment by kapilkd13** [comment]
Hi @henrykironde I cleaned my branch with `git clean -xf`. Only extra file  I got was version.txt. Removed it.

**Comment by ethanwhite** [comment]
Thanks for this @kapilkd13. My main comment is that we don't need to have a message for every dataset. The idea is that in special cases we may want to print a message, so we should have the ability to do so, but it should only be used if needed. So, in this PR we should take out all of the additions to the existing scripts and the code in `datapackage.py` that generates this for new scripts.

**Comment by kapilkd13** [comment]
Hi @ethanwhite  I have removed message field from existing JSON's. In new scripts what I am thinking is, we will ask a question like `Want to add a message[y/n]` if user pressed responded Yes, then only we will add message field to it, else not. Is this approach good. Thanks.

**Comment by ethanwhite** [comment]
> In new scripts what I am thinking is, we will ask a question like Want to add a message[y/n] if user pressed responded Yes, then only we will add message field to it, else not. Is this approach good. Thanks.

Yes, that's perfect. Thanks!

**Comment by kapilkd13** [comment]
Hi, I have added option to put special message in `new_json` if user require it, as per your suggestion. Thanks.

File: retriever/lib/templates.py [hunk 1]
```diff
@@ -112,6 +114,10 @@ def reference_url(self):
             if len(self.urls) == 1:
                 return '/'.join(self.urls[list(self.urls.keys())[0]].split('/')[0:-1]) + '/'
 
+    def printMessage(self):
+        if self.message!="":
```
Comment (ethanwhite) [inline retriever/lib/templates.py]: `""` evaluates to `False` in Python, so the Pythonic form of this conditional is `if self.message:`

**Comment by ethanwhite** [comment]
Oh, and it looks we we need to rebase due to changes in `lib/templates.py`. If that's too much trouble to deal with just let us know and we can always merge this locally.

**Comment by kapilkd13** [comment]
Hi @ethanwhite I made changes you suggested. Also rebased it on latest commits. I think it's ready to merge.

**Comment by ethanwhite** [comment]
Awesome! Thanks @kapilkd13! In it goes.

**Pull request merged**